com.chartboost.helium/Editor/HeliumIntegrationChecker.cs

```diff
@@ -0,0 +1,329 @@
+        {
+            Directory.Delete(HeliumSamplesInAssets, true);
+            File.Delete(HeliumSamplesMetaInAssets);
+            AssetDatabase.Refresh();
+
+            var allSamples = Sample.FindByPackage(HeliumPackageName, version);
+
+            foreach (var sample in allSamples)
+            {
+                if (existingSamples.Contains(sample.displayName))
+                    sample.Import(ImportOptions.HideImportWindow | ImportOptions.OverridePreviousImports);
+            }
+        }
+
+        /// <summary>
+        /// Used to attempt to update all existing Ad Adapters without extra input. Good for CI/CD usage and update of Adapters.
+        /// </summary>
+        /// <returns>Update attempt status</returns>
+        public static bool ReimportExistingAdapters()
+        {
+            var helium = FindPackage(HeliumPackageName);
+
+            if (!Directory.Exists(HeliumSamplesInAssets))
+                return false;
+
+            var subdirectories = Directory.GetDirectories(HeliumSamplesInAssets);
+            if (subdirectories.Length <= 0)
+                return false;
+
+            var versionDirectory = subdirectories[0];
+            var importedDependencies = new HashSet<string>();
+            // find all samples/ad adapters
+            foreach (var imported in Directory.GetDirectories(versionDirectory))
+            {
+                var sampleName = Path.GetFileName(imported);
+                importedDependencies.Add(sampleName);
+            }
+
+            ReimportExistingHeliumSamples(importedDependencies, helium.version);
+            return true;
+        }
+
+        /// <summary>
+        /// Uncomment UnityAds dependency on Optional-HeliumUnityAdsDependencies.xml if present.
+        /// </summary>
+        /// <param name="skipDialog">Optional parameter to skip any dialog windows, if true, UnityAds dependency will be uncommented if possible.</param>
+        public static void UncommentUnityAdsDependency(bool skipDialog = false)
+        {
+            var helium = FindPackage(HeliumPackageName);
+            if (!Version.TryParse(helium.version, out var heliumFoundVersion))
+            {
+                if (!skipDialog)
+                {
+                    EditorUtility.DisplayDialog(
+                        HeliumWindowTitle,
+                        $"Failed to parse version {helium.version} in Package.\n\n**This is probably a bad setup, contact Helium Support at support@chartboost.com**",
+                        "Ok");
+                }
+                return;
+            }
+
+            var heliumVersion = heliumFoundVersion.ToString();
+            var unityAdsDependencyPath = $"Assets/Samples/Helium SDK/{heliumVersion}/UnityAds/Editor/Optional-HeliumUnityAdsDependencies.xml";
+
+            // check if UnityAds is integrated
+            if (!File.Exists(unityAdsDependencyPath))
```

Comment:
Do most publishers make the attempt to use Unity Ads?? Rather than making Unity Ads a crucial part of Helium integrations, can we make it optional and print it in the console as a reminder?

Reply:
@cb-jpadilla this will only happen if UnityAds exists, it is optional, and this will always be optional. This check is just in place for those who do integrate it, to be able to double check that they are at least following our bare minimum. Let me know if you need info.